components/director/cmd/director/main.go

```diff
@@ -222,3 +235,40 @@ func getTenantMappingHanderFunc(transact persistence.Transactioner, staticUsersS
 
 	return tenantmapping.NewHandler(reqDataParser, transact, mapperForUser, mapperForSystemAuth).ServeHTTP, nil
 }
+
+func getRuntimeMappingHanderFunc(transact persistence.Transactioner, cachePeriod time.Duration, stopCh <-chan struct{}) (func(writer http.ResponseWriter, request *http.Request), error) {
+	logger := log.WithField("component", "runtime-mapping-handler").Logger
+
+	uidSvc := uid.NewService()
+
+	labelConv := label.NewConverter()
+	labelRepo := label.NewRepository(labelConv)
+	labelDefConverter := labeldef.NewConverter()
+	labelDefRepo := labeldef.NewRepository(labelDefConverter)
+	scenariosSvc := labeldef.NewScenariosService(labelDefRepo, uidSvc)
+	labelUpsertSvc := label.NewLabelUpsertService(labelRepo, labelDefRepo, uidSvc)
+	runtimeRepo := runtime.NewRepository()
+	runtimeSvc := runtime.NewService(runtimeRepo, labelRepo, scenariosSvc, labelUpsertSvc, uidSvc)
+
+	tenantConv := tenant.NewConverter()
+	tenantRepo := tenant.NewRepository(tenantConv)
+	tenantSvc := tenant.NewService(tenantRepo, uidSvc)
+
+	reqDataParser := tenantmapping.NewReqDataParser()
```

Comment:
Just a thought but I wonder if Request Data Parser and models it uses shouldn't be moved out of `tenantmapping` package, it seems that they're equally related to tenant and runtime mapping now. What do you think?

Reply:
done